### PR TITLE
[PWGCF] add topological cut QA and flow uncertainties

### PR DESCRIPTION
**Pull request by fuchuncui**
Add and change some cuts for event selection and topological cut
Add switch of daughter TOF and TPC check
Add V0 cuts QA
Add uncertainties using Jackknife method (10 sub-samples)

### CODE REVIEW

File: PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx [hunk 1]
```diff
@@ -557,24 +680,44 @@ struct FlowGFWOmegaXi {
       auto v0negdau = v0.negTrack_as<DaughterTracks>();
       // check tpc
       int PDGCode = 0;
-      if (v0.qtarm() / TMath::Abs(v0.alpha()) > cfgv0_ArmPodocut && TMath::Abs(v0posdau.tpcNSigmaPi()) < cfgNSigmaCascPion && TMath::Abs(v0negdau.tpcNSigmaPi()) < cfgNSigmaCascPion && TMath::Abs(v0posdau.tofNSigmaPi()) < cfgNSigmaCascPion && TMath::Abs(v0negdau.tofNSigmaPi()) < cfgNSigmaCascPion) {
+      // fill QA
+      registry.fill(HIST("hqaarm_podobefore"), v0.alpha(), v0.qtarm());
+      // check daughter TPC and TOF
+      if (v0.qtarm() / TMath::Abs(v0.alpha()) > cfgv0_ArmPodocut &&
+          (!cfgcheckDauTPC || (TMath::Abs(v0posdau.tpcNSigmaPi()) < cfgtpcNSigmaCascPion && TMath::Abs(v0negdau.tpcNSigmaPi()) < cfgtpcNSigmaCascPion)) &&
+          (!cfgcheckDauTOF || (TMath::Abs(v0posdau.tofNSigmaPi()) < cfgtofNSigmaCascPion && TMath::Abs(v0negdau.tofNSigmaPi()) < cfgtofNSigmaCascPion))) {
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx:688]: It would be good to start moving to `std::fabs`
O2Physics workflows are `c++` not `ROOT` programs

**Review by victor-gonzalez — APPROVED** [review]
Please, consider my comment for further iterations

File: PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx (hunk 1 above)
Comment (fuchuncui) [reply PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx]: Thanks for your comment.

**Review by victor-gonzalez — APPROVED** [review]
Thanks!

**Pull request merged**